corehq/apps/domain/forms.py

```diff
@@ -315,6 +315,13 @@ class DomainGlobalSettingsForm(forms.Form):
     )
     default_timezone = TimeZoneChoiceField(label=ugettext_noop("Default Timezone"), initial="UTC")
 
+    default_geocoder_location = Field(
+        widget=GeoCoderInput(attrs={'placeholder': ugettext_lazy('Select a location')}),
+        label=ugettext_noop("Default geocoder location"),
+        required=False,
+        help_text=ugettext_lazy("Please select the default location for the geocoder used in your project space.")
```

Comment:
As we discussed, I think this wording isn't totally clear. I left a comment on the spec to get input from Kat.